src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -170,6 +170,8 @@
     "span.main_thread": PREFIX + 255,
     "device.class": PREFIX + 256,
     "resource.render_blocking_status": PREFIX + 257,
+    "ttid": PREFIX + 257,
```

Comment:
`257` is already on the above line, did you want to use it with `258` and next one `259`?

Reply:
LOL sorry for that. Thankfully we have validation for this.